charts/kubewarden-controller/values.yaml

```diff
@@ -227,6 +227,18 @@ resources:
     requests:
       cpu: 250m
       memory: 300Mi
+mTLS:
+  # Enable mutual TLS authentication. This will require TLS connections with both
+  # the server and client being authenticated, between the kubewarden
+  # policy-servers and the Kubernetes API server, as well as between the
+  # policy-servers and the audit-scanner.
+  # Enabling this feature will require the Kubernetes API server to be
+  # configured. If that is not achievable, consider using a service mesh. 
+  enable: false
+  # name of the ConfigMap in kubewarden-controller namespace containing the
+  configMapName: ""
+  # client CA certificate. The ConfigMap must contain a data.client-ca.crt key
```

Comment:
It would be cleaner to move comments above configMapName, now comments are split in half by value.
Otherwise I tested it yesterday and it worked fine.

Reply:
This is weird. Locally, I have the configMapName set to a string for testing below the code block, and when stashing the comment block it took it up. Fixed now.